schema-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaSchemaSerDeConfig.java

```diff
@@ -167,7 +179,13 @@ public static ConfigDef baseConfigDef() {
         .define(PROXY_HOST, Type.STRING, PROXY_HOST_DEFAULT,
                 Importance.LOW, PROXY_HOST_DOC)
         .define(PROXY_PORT, Type.INT, PROXY_PORT_DEFAULT,
-                Importance.LOW, PROXY_PORT_DOC);
+                Importance.LOW, PROXY_PORT_DOC)
+        .define(SCHEMA_REGISTRY_HTTP_CONNECT_TIMEOUT_MS, Type.INT,
+            SCHEMA_REGISTRY_HTTP_CONNECT_TIMEOUT_MS_DEFAULT, Importance.LOW,
+            SCHEMA_REGISTRY_HTTP_CONNECT_TIMEOUT_MS_DOC)
+        .define(SCHEMA_REGISTRY_HTTP_READ_TIMEOUT_MS, Type.INT,
```

Comment:
@rayokota , here we are making sure the type is of the Int, so I think we can be sure it can't be a string.

Reply:
> @rayokota , here we are making sure the type is of the Int, so I think we can be sure it can't be a string.

I think the type coercion is only performed if you call `getInt(..)` on AbstractKafkaSchemaSerDeConfig, which is not what we're doing in RestService.